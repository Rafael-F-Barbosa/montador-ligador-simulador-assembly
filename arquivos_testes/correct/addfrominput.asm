            section data
first_op:   SPACE
second_op:  SPACE
result:     SPACE

            section text
start:                                      ; This is a beautifully formatted code
            INPUT   first_op                ; Read first operand
            INPUT   second_op               ; Read seconde operand
            LOAD    first_op                ; Load first operand into ACC
            ADD     second_op               ; ACC <- ACC + second operand
            STORE   result                  ; Store result for output
            OUTPUT  result                  ; Show result
            STOP
