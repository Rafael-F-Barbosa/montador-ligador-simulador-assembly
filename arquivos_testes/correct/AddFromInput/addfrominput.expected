> 2
PC <- 2
ACC <- 0

> 5
PC <- 4
ACC <- 0

PC <- 6
ACC <- 2

PC <- 8
ACC <- 7

PC <- 10
ACC <- 7

PC <- 12
ACC <- 7
OUTPUT: 7

PC <- 13
ACC <- 7
